File: Java/aprovados.java
import java.util.Locale;
import java.util.Scanner;

public class aprovados {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n;
		double 	media;
		
		System.out.print("Quantos alunos serao digitados? ");
		n = sc.nextInt();
		
		String[] alunos = new String[n];
		double[] nota1 = new double[n];
		double[] nota2 = new double[n];
		
		for (int i = 0; i < n; i++) {
			System.out.println("Digite nome, primeira e segunda nota do "+ (i + 1) +"o aluno:");
			sc.nextLine();
			alunos[i] = sc.nextLine();
			nota1[i] = sc.nextDouble();
			nota2[i] = sc.nextDouble();
		}
		
		System.out.println("Alunos aprovados:");
		
		for (int i = 0; i < n; i++) {			
			media = (nota1[i] + nota2[i]) / 2;
			if (media >= 6.0) {
				System.out.println(alunos[i]);
			}
		}
		
		sc.close();
	}	
}

File: Python/aprovados.py
media: float
n = int(input("Quantos alunos serao digitados? "))

alunos = [0 for x in range(n)]
notas_1sem = [0 for x in range(n)]
notas_2sem = [0 for x in range(n)]

for i in range(0, n):
    print(f"Digite nome, primeira e segunda nota do {i + 1}o aluno:")
    alunos[i] = str(input())
    notas_1sem[i] = float(input())
    notas_2sem[i] = float(input())

print("Alunos aprovados:")
soma_notas = 0
for i in range(0, n):
    media = (notas_1sem[i] + notas_2sem[i]) / 2.0
    if media >= 6.0:
        print(alunos[i])